applications/doodle/src/doodle.erl

```diff
@@ -24,6 +25,16 @@
 start_link() ->
     _ = start_deps(),
     _ = declare_exchanges(),
```

Comment:
can you explain this to me?  Are you trying to update the existing doodle config doc with the reschedule object?